lib/widgets/compose_box.dart

```diff
@@ -1368,10 +1418,18 @@ class StreamComposeBoxController extends ComposeBoxController {
   final ComposeTopicController topic;
   final topicFocusNode = FocusNode();
 
+  /// Whether the user has made up their mind choosing a topic.
+  ///
+  /// Empirically, this should be set to `false` whenever the user focuses on
+  /// the topic input, and set to `true` whenever the user focuses on the
+  /// content input.
+  ValueNotifier<bool> hasChosenTopic = ValueNotifier(false);
```

Comment:
How about "Whether the user seems to have made up their mind choosing a topic." or similar? It's impossible to know for sure, right :) they could come back and change it again before sending, or they could send accidentally before making up their mind.

Also I don't understand what "empirically" means here. The sentence sounds like we're an observer making notes from what we've seen of the app's behavior, rather than the ones deciding what the behavior is and causing it. 🙂